doc/source/catalog_generation.rst

```diff
@@ -436,6 +530,241 @@ good and the other contaminated.
 Should the catalogs fail this test, neither type of catalogs will be written out to disk for this visit.
 
 
-Segment Photometric Catalog Generation
-=======================================
-Michele's documentation goes here!
+4: Segmentation Catalog Generation
+----------------------------------
+
+4.1: Source Identification with PhotUtils
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For the segmentation algorithm the
+`photutils.segmentation <https://photutils.readthedocs.io/en/stable/segmentation.html>`_ Astropy 
+tool is used to identify sources in the background-subtracted multi-filter detection image. 
+To identify a signal as a source, the signal must have a minimum number
+of connected pixels, each of which is greater than its two-dimensional threshold image
+counterpart.  Connectivity refers to how pixels are literally touching along their edges and 
+corners, and the threshold image is the background RMS image (Section 1.3) 
+multiplied by a configurable n-sigma value and modulated by a weighting scheme based
+upon the WHT extension of the detection image. Before applying the threshold, the detection 
+image is filtered by the image kernel (Section 1.4) to smooth the data and enhance the ability 
+to identify signal which is similar in shape to the kernel. This process generates a two-dimensional
+segmentation image or map where a segment is defined to be a number of connected pixels which are
+all identified by a numeric label and are considered part of the same source. 
+
+Because different sources in close proximity can be mis-identified as a single source, it is necessary
+to apply a deblending procedure to the segmentation map.  The deblending is a combination of 
+multi-thresholding, as is done by `Source Extractor <https://sextractor.readthedocs.io/en/latest/Introduction.html>`_
+and the `watershed technique <https://en.wikipedia.org/wiki/Watershed_(image_processing)>`_.
+The deblending can be problematic if the background determination has not been well-determined,
+resulting in segments which are a large percentage of the map footprint.  In this case, the
```

Comment:
The sentence regarding taking so long to deblend might warrant another admonition; perhaps, 

.. caution::

This is something that can definitely impact the end user or even the pipeline, and should be highlighted as a potential problem in processing the data.

Reply:
Fixed and will be pushed with other fixes which highlight certain text.